cloudify_rest_client/deployments.py

```diff
@@ -256,13 +256,21 @@ def put(self, group_id, visibility=VisibilityState.TENANT,
         :param description: description of the group
         :param blueprint_id: the default blueprint to use when extending
         :param default_inputs: the default inputs to use when extending
-        :param deployment_ids: set the group deployments to these
+        :param deployment_ids: make the group contain these
+            existing deployments
         :param filter_id: set the group to contain the deployments matching
-                          this filter
-        :param inputs: create new deployments using these inputs merged
-                       with default_inputs, and using default_blueprint
+            this filter
+        :param new_deployments: create new deployments using this
+            specification, merged with the group's default_blueprint
+            and default_inputs
+        :type new_deployments: a list of dicts, each can contain the
+            key "inputs"
```

Comment:
... which will be used to populate the inputs of the new deployment?

Reply:
After some discussion, I think we should stick with this for the moment and see if we need to get more descriptive when more fields added.